Source/Core/Common/Random.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/Random.h"
+
+#include <mbedtls/entropy.h>
+#include <mbedtls/hmac_drbg.h>
+
+#include "Common/Assert.h"
+
+namespace Common::Random
+{
+class CSPRNG final
+{
+public:
+  CSPRNG()
+  {
+    mbedtls_entropy_init(&m_entropy);
+    mbedtls_hmac_drbg_init(&m_context);
+    const int ret = mbedtls_hmac_drbg_seed(&m_context, mbedtls_md_info_from_type(MBEDTLS_MD_SHA256),
+                                           mbedtls_entropy_func, &m_entropy, nullptr, 0);
+    ASSERT(ret == 0);
```

Comment:
Is this a run-time assert or debug only?

Reply:
Runtime. DEBUG_ASSERT is the debug only version.